### ISSUE TITLE
Need help on "multi_bw_min" in mgwr.sel_bw

### ISSUE THREAD

**Issue by Frank-Fy** [pysal/mgwr#76]
- I had trouble learning the mgwr package. Could you help me?
- As in help document: **multi_bw_min** is a list of min values used for each covariate in mgwr bandwidth search. Must be either a single value or have one value for each covariate including the intercept.

1. How should I understand the minimum of covariates? 
2. And why set it to 2 in the following example?

- When I set the multi_bw_min to zero in my cases, the adjust R square is 0.82, while setting it to "None" the R square is 0.49.

Follwing is the example in help document: 
 

>     |  --------
>      |  
>      |  #basic model calibration
>      |  
>      |  >>> import libpysal as ps
>      |  >>> from mgwr.gwr import MGWR
>      |  >>> from mgwr.sel_bw import Sel_BW
>      |  >>> data = ps.io.open(ps.examples.get_path('GData_utm.csv'))
>      |  >>> coords = list(zip(data.by_col('X'), data.by_col('Y')))
>      |  >>> y = np.array(data.by_col('PctBach')).reshape((-1,1))
>      |  >>> rural = np.array(data.by_col('PctRural')).reshape((-1,1))
>      |  >>> fb = np.array(data.by_col('PctFB')).reshape((-1,1))
>      |  >>> african_amer = np.array(data.by_col('PctBlack')).reshape((-1,1))
>      |  >>> X = np.hstack([fb, african_amer, rural])
>      |  >>> X = (X - X.mean(axis=0)) / X.std(axis=0)
>      |  >>> y = (y - y.mean(axis=0)) / y.std(axis=0)
>      |  >>> selector = Sel_BW(coords, y, X, multi=True)
>      |  >>> selector.search(multi_bw_min=[2])
>      |  [92.0, 101.0, 136.0, 158.0]
>      |  >>> model = MGWR(coords, y, X, selector, fixed=False, kernel='bisquare', sigma2_v1=True)
>      |  >>> results = model.fit()
>      |  >>> print(results.params.shape)
>      |  (159, 4)
